cmd/thanos/store.go

```
@@ -213,6 +213,7 @@ func runStore(
 	if err != nil {
 		return err
 	}
+	confContentYamlStr, _ := config.ReplaceSecret(confContentYaml)
```

Don't just ignore errors.